### PR TITLE
#1026 Add versions to all plugins

### PR DESCRIPTION
**Pull request by 3rdvision**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-white/issues/1026 |
| Decisions | Add missing versions to all plugins. This is going to be used to get all technical plugin versions [here](https://github.com/ripe-tech/ripe-white/pull/1031/files#diff-0011c60ccfe7d6c1e67fe3d7f942c9da4554d930ac92c8d8383a1e3e7f8128feR270) |

### CODE REVIEW

**Review by joao-conde — APPROVED** [review]

**Review by BeeMargarida — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-*
+* Added version to all plugins starting at 0.1.0 [ripe-white/#1026](https://github.com/ripe-tech/ripe-white/issues/1026)
```
Comment (BeeMargarida) [inline CHANGELOG.md:12]:
```suggestion
* Added version to all plugins starting at 0.1.0 - [ripe-white/#1026](https://github.com/ripe-tech/ripe-white/issues/1026)
```

Comment (3rdvision) [reply CHANGELOG.md:12]: :eye:

**Comment by joamag** [comment]
This may seem weird but the expected way of changing plugin's version is to override the getVersion method

**Comment by joamag** [comment]
Do that before the getDescription one

**Comment by 3rdvision** [comment]
> This may seem weird but the expected way of changing plugin's version is to override the getVersion method

There's already the `getVersion` plugin method which returns `this.version`
image

Since we have constructors using this strategy of defining the version prior to this change like for [example](https://github.com/ripe-tech/ripe-vendors-unspun/blob/master/index.js#L8). I just thought it would be less lines of code and more consistent to keep it that way than having some plugins defining versions in this.version and other overriding the method getVersion

**Comment by joamag** [comment]
@3rdvision it is weird but if you look at `getDescription()` it's exactly the same situation. We opted to go with the override of methods for some reason...

**Comment by joamag** [comment]
Sorry for complicating things but we need to fix the order of method definition to respect this one :(

image

**Comment by 3rdvision** [comment]
> Sorry for complicating things but we need to fix the order of method definition to respect this one :(

Done

**Review by joamag — APPROVED** [review]

**Pull request merged**